source/direct_debit/index.html.md.erb

```diff
@@ -0,0 +1,273 @@
+---
+title: Direct Debit
+weight: 75
+---
+
+# GOV.UK Pay Variable Direct Debit
+
+Before reading this section, you should be familiar with the [__Making
+payments__ section](/payment_flow_overview/#payment-flow). You should also
+have [integrated your service with the GOV.UK Pay
+API](/integrate_with_govuk_pay/#how-to-integrate-with-the-gov-uk-pay-api).
+
+To use GOV.UK Pay for variable Direct Debit transactions, you should:
+
+* request a GOV.UK Pay Direct Debit test account [from our
+support team](/support_contact_and_more_information/#contact-us).
+
+* add support for the Direct Debit endpoints to your existing GOV.UK Pay
+integration
+
+* sign in to your GOV.UK Pay Direct Debit account and follow the instructions
+to set up an account with our Direct Debit supplier
+
+## GOV.UK Pay ‘agreements’
+
+In order to support Direct Debit, GOV.UK Pay now uses ‘agreements’. An
```

Comment:
slight hesitation about using 'now' as it introduces temporality - but not too worried since i'm sure we'll be updating the docs when the feature is less new

Reply:
Agree, was hoping to get rid of this anyway but wanted to double-check how you wanted to pitch it